packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -277,6 +280,16 @@ abstract class FlutterCommand extends Command<void> {
     _usesTargetOption = true;
   }
 
+  void usesFatalLogOutputOption({ @required bool verboseHelp }) {
+    argParser.addFlag(FlutterOptions.kFatalLogWarnings,
+        defaultsTo: false,
+        hide: !verboseHelp,
+        help: 'Causes the command to fail if warnings are sent to the log '
```

Comment:
log -> console?

Reply:
OK, I renamed this. Really, `log` is more accurate, since the log output doesn't necessarily go to the console, but I agree that for a user reading the help message, it's 99.99% of the time going to be the console, and saying "log" makes them wonder where the log file is.